### ISSUE TITLE
Global Header - No label on link (537)

### ISSUE THREAD

**Issue by jpante** [Workday/canvas-kit#512]
## To Reproduce

Steps to reproduce

1. Navigate to https://workday.github.io/canvas-kit/?path=/story/labs-header-react--global-header
2. Turn on NVDA
3. Press tab until focus is on the banner Workday logo image link

## Expected Behavior

Reads "Workday Home link".

## Actual Results

Reads "iframe id=labs-header-react--g…
To get missing image descriptions, open the context menu.
link"

## Suggested fix

Put aria-label="Workday Home" on either source image or link and remove additional label text which seems to be dynamically added in.

**Comment by vibdev** [comment]
Looks like the image is missing an alt tag. `modules/common/react/lib/parts/_brand-assets.ts` doesn't have any. Should the header have an image slot instead to support non workday logos?

**Comment by NicholasBoll** [comment]
@vibdev I think it makes sense to require a logo to be passed in rather than defaulting to the Workday logo. Perhaps a logo slot rather than an image slot. Someone may provide the SVG instead of an `img` tag of an `SVG` source. Our stories could then just slot in the Workday logo.
